Classes/AssetUsageIndex.php

```diff
@@ -198,7 +198,7 @@ public function extractAssetIdsFromPropertyValue(string $type, $value): array
         if ($value instanceof ResourceBasedInterface) {
             return [$this->getAssetId($value)];
         }
-        if ($type === 'string' || is_subclass_of($type, Stringable::class)) {
+        if (is_string($value) || is_subclass_of($type, Stringable::class)) {
```

Comment:
I think it makes sense to ignore the `type` definition here, but we should do so in both cases I think:
```suggestion
        if (is_string($value) || $value instanceof Stringable) {
```

btw: we do the same in the Neos 9 built-in [AssetUsageProjection](https://github.com/neos/neos-development-collection/blob/9cd8c4f790955a4d69e7e4baa59a35d5b434e422/Neos.Neos/Classes/AssetUsage/Projection/AssetUsageProjection.php#L204-L207)  (it's a bit simpler because values can only be scalars there)